packages/api-explorer/src/components/DocPseudo/DocParams.tsx

```diff
@@ -9,12 +33,12 @@ export interface DocArgsProps {
 }
 
 export const DocParams: FC<DocArgsProps> = ({ parameters }) => (
-  <Flex flexWrap="wrap">
+  <span>
     {parameters.map((a, index) => (
-      <FlexItem key={index}>
+      <span key={index}>
```

Comment:
In this case you can also use fragment but you have to use the non-shortcode version to add the index:

```suggestion
      <Fragment key={index}>
```

Reply:
Note that you'll need to add `Fragment` to the react import above.